sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -148,6 +153,10 @@ export class CryptographyClient {
       }
     }
 
+    // Renaming parameters
+
+    requestOptions.aad = options.additionalAuthenticatedData;
```

Comment:
why do we have to do this? Feels like a swagger transform could make the generated client take the better name.

Reply:
I'm not sure how to work with swagger transforms! I'd like to learn 🌞

Reply:
Thank you! I'll try with something like this: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/swagger/README.md

Reply:
I did the change in two parts:

- https://github.com/Azure/azure-sdk-for-js/pull/11380/commits/e155811be5e3dc205777aa44e48aa11e5b92cf4a
- https://github.com/Azure/azure-sdk-for-js/pull/11380/commits/872e59736a42cb513ede3d85268e1a4e9f30b982

Please let me know how it looks 🌞

Reply:
Nice! LGTM